src/components/Sing/ScoreSequencer.vue

```diff
@@ -1,23 +1,349 @@
+            v-bind:height="`${BASE_Y_SIZE * zoomY}`"
+            v-bind:fill="`${y.color === 'white' ? '#fff' : '#555'}`"
+          />
+          <line
+            x1="0"
+            v-bind:y1="`${(index + 1) * BASE_Y_SIZE * zoomY}`"
+            x2="64"
+            v-bind:y2="`${(index + 1) * BASE_Y_SIZE * zoomY}`"
+            v-bind:stroke="`${
+              y.pitch === 'C' ? '#bbb' : y.pitch === 'F' ? '#ddd' : '#fff'
+            }`"
+            stroke-width="1"
+          />
+          <text
+            font-size="10"
+            fill="#555"
+            x="48"
+            v-bind:y="`${BASE_Y_SIZE * zoomY * (index + 1) - 4}`"
+            v-bind:opacity="y.pitch === 'C' ? 1 : 0"
+          >
+            {{ y.name }}
+          </text>
+        </g>
+      </svg>
+    </div>
+    <div
+      class="sequencer-grids"
+      v-bind:style="{
+        height: `${BASE_Y_SIZE * zoomY * 128 + 164}px`,
+        width: `${gridX.length * BASE_X_SIZE * zoomX}px`,
+      }"
+    >
+      <svg
+        width="100%"
+        height="100%"
+        xmlns="http://www.w3.org/2000/svg"
+        v-bind:style="{
+          position: 'relative',
+          top: 0,
+          left: 0,
+        }"
+      >
+        <defs>
+          <pattern
+            id="grid"
+            v-bind:width="`${BASE_X_SIZE * zoomX}px`"
+            v-bind:height="`${12 * BASE_Y_SIZE * zoomY}px`"
+            patternUnits="userSpaceOnUse"
+          >
+            <rect
+              v-for="(y, index) in gridY"
+              :key="index"
+              x="0"
+              v-bind:y="`${BASE_Y_SIZE * zoomY * index}`"
+              v-bind:width="`${BASE_X_SIZE * zoomX}`"
+              v-bind:height="`${BASE_Y_SIZE * zoomY}`"
+              v-bind:fill="`${y.color === 'white' ? '#fff' : '#eee'}`"
+              stroke-width="1"
+              stroke="#ddd"
+            />
+          </pattern>
+        </defs>
+        <rect
+          x="0"
+          y="0"
+          width="100%"
+          height="100%"
+          id="grid"
+          fill="url(#grid)"
+          @click="(e) => addNote(e)"
+        />
+      </svg>
+      <!-- NOTE: ノートと歌詞入力あわせコンポーネント分割予定 -->
+      <div
+        v-for="(note, index) in notes"
+        :key="index"
+        class="sequencer-note"
+        v-bind:style="{
+          left: `${(note.position / 4) * zoomX}px`,
+          top: `${(127 - note.midi) * BASE_Y_SIZE * zoomY}px`,
+        }"
+      >
+        <input
+          type="text"
+          :value="note.lyric"
+          @input="(e) => setLyric(index, e)"
+          class="sequencer-note-lyric"
+          v-bind:style="{ bottom: `${BASE_Y_SIZE * zoomY + 4}px` }"
+        />
+        <svg
+          v-bind:height="`${BASE_Y_SIZE * zoomY}`"
+          v-bind:width="`${(note.duration / 4) * zoomX}`"
+          xmlns="http://www.w3.org/2000/svg"
+          v-bind:style="{
+            position: 'relative',
+            top: 0,
+            left: 0,
+          }"
+        >
```

Comment:
zoomYの値が小さい時にノートが下にずれてしまっているみたいです。
svgはデフォルトではインライン要素になるらしく、svgに`display: block`を追加するとずれなくなると思います！
また、inputのbottomは100%にしても良いかもです！
```suggestion
        <input
          type="text"
          :value="note.lyric"
          @input="(e) => setLyric(index, e)"
          class="sequencer-note-lyric"
          v-bind:style="{ bottom: '100%' }"
        />
        <svg
          v-bind:height="`${BASE_Y_SIZE * zoomY}`"
          v-bind:width="`${(note.duration / 4) * zoomX}`"
          xmlns="http://www.w3.org/2000/svg"
          v-bind:style="{
            display: 'block',
            position: 'relative',
            top: 0,
            left: 0,
          }"
        >
```

Reply:
@sigprogramming 
おおお、ありがとうございます！見落としておりました。